larray/core/array.py

```diff
@@ -552,97 +552,17 @@ def _doc_agg_method(func, by=False, long_name='', action_verb='perform', extra_a
-    nan_equals: boolean, optional
-        Whether or not to consider nan values at the same positions in the two arrays as equal.
-        By default, an array containing nan values is never equal to another array, even if that other array
-        also contains nan values at the same positions. The reason is that a nan value is different from
-        *anything*, including itself. Defaults to False.
-
-    Returns
-    -------
-    LArray
-        Boolean array of where a1 and a2 are equal within a tolerance range if given.
-        If nan_equals=True, nan’s in a1 will be considered equal to nan’s in a2 in the output array.
-
-    Notes
-    -----
-    For finite values, element_equal uses the following equation to test whether two values are equal:
-
-        absolute(array1 - array2) <= (atol + rtol * absolute(array2))
-
-    The above equation is not symmetric in array1 and array2, so that element_equal(array1, array2)
-    might be different from element_equal(array2, array1) in some rare cases.
-
-    Examples
-    --------
-    >>> arr1 = LArray([6., np.nan, 8.], "a=a0..a2")
-    >>> arr1
-    a   a0   a1   a2
-       6.0  nan  8.0
-
-    Default behavior (same as == operator)
-
-    >>> element_equal(arr1, arr1)
-    a    a0     a1    a2
-       True  False  True
-
-    Test equality between two arrays within a given tolerance range.
-    Return True if absolute(array1 - array2) <= (atol + rtol * absolute(array2)).
-
-    >>> arr2 = LArray([5.999, np.nan, 8.001], "a=a0..a2")
-    >>> arr2
-    a     a0   a1     a2
-       5.999  nan  8.001
-    >>> element_equal(arr1, arr2, nan_equals=True)
-    a     a0    a1     a2
-       False  True  False
-    >>> element_equal(arr1, arr2, atol=0.01, nan_equals=True)
-    a    a0    a1    a2
-       True  True  True
-    >>> element_equal(arr1, arr2, rtol=0.01, nan_equals=True)
-    a    a0    a1    a2
-       True  True  True
-    """
-    a1, a2 = aslarray(a1), aslarray(a2)
-
-    if rtol == 0 and atol == 0:
-        if not nan_equals:
-            return a1 == a2
-        else:
-            from larray.core.ufuncs import isnan
-
-            def general_isnan(a):
-                if np.issubclass_(a.dtype.type, np.inexact):
-                    return isnan(a)
-                elif a.dtype.type is np.object_:
-                    return LArray(obj_isnan(a), a.axes)
-                else:
-                    return False
-
-            return (a1 == a2) | (general_isnan(a1) & general_isnan(a2))
-    else:
-        (a1, a2), res_axes = make_numpy_broadcastable([a1, a2])
-        return LArray(np.isclose(a1.data, a2.data, rtol=rtol, atol=atol, equal_nan=nan_equals), res_axes)
-
-element_equal = renamed_to(element_equals, 'element_equal')
+def element_equal(a1, a2, rtol=0, atol=0, nan_equals=False):
+    import warnings
+    warnings.warn("element_equal() is deprecated. Use LArray.eq() instead.", FutureWarning, stacklevel=2)
+    a1 =aslarray(a1)
```

Comment:
missing space. didn't pep8 check flag this?

Reply:
I started this PR some time ago. PEP8 is not yet activated but will be as soon as I rebase.